lib/ansible/modules/import_role.py

```diff
@@ -56,6 +56,14 @@
     type: bool
     default: yes
     version_added: '2.11'
+  public:
+    description:
+      - This option dictates whether the role's C(vars) and C(defaults) are exposed to the play.
+      - Variables are exposed to the play at playbook parsing time, and available to earlier roles and tasks as well unlike C(include_role).
+      - The default depends on the configuration option C(DEFAULT_PRIVATE_ROLE_VARS) configuration option.
+    type: bool
+    default: yes
```

Comment:
I kind of feel like this is overloading `public` since the bug is related to *when* exporting happens, not whether it happens.

Reply:
true, so I"m leaning to leaving it as a binary for "if" it happens, letting import behave differently than include for "when".